apps/web/src/pages/integrations/IntegrationsStoreModal.tsx

```diff
@@ -39,7 +39,7 @@ export function IntegrationsStoreModal({
   const { environment } = useEnvController();
   const { organization } = useAuthController();
   const { loading: isLoading } = useIntegrations({ refetchOnMount: false });
-  const { emailProviders, smsProvider, chatProvider, pushProvider } = useProviders();
+  const { emailProviders, smsProvider, chatProvider, pushProvider, inAppProvider } = useProviders();
```

Comment:
I guess that the different in the pluralisation is that we allow multiple providers now because of our free email provider. Am I right?

Reply:
Not sure why we call it providers in case of email...